component-generator/index.js

```diff
@@ -0,0 +1,23 @@
+const { program } = require('commander');
+const fs = require('fs');
+const { exec } = require('child_process');
+const path = require('path');
+const { COMPONENT_FILES } = require('./constants');
+const { validateComponentName, createFile, addComponentToExports } = require('./utils');
+
+program
+  .requiredOption('--componentName <name>', 'Component must have a name', validateComponentName)
+  .action((options) => {
+    const { componentName } = options;
+    const componentDir = path.resolve(__dirname, `../src/${componentName}`);
+    // create directory for the component files
+    fs.mkdirSync(componentDir);
+    // create all necessary files for the component
+    COMPONENT_FILES.forEach(file => createFile(file.targetPath, file.templatePath, componentName));
+    // export component and its styles from Paragon
+    addComponentToExports(componentName);
+    // add generated files to git
+    exec(`git add ${componentDir}/*`);
```

Comment:
I think adding only the files in the `componentDir` results in not adding the changes made to the exports via `addComponentToExports`. After running the command, I see the 2 files in `src` unadded by Git.

![image](https://user-images.githubusercontent.com/2828721/148421686-8f3393f3-271e-4ca3-95d8-5e64798c56a3.png)

Reply:
Right, somehow I missed that... Updated to add exports also, thanks!